lumicks/pylake/nb_widgets/kymotracker_widgets.py

```diff
@@ -603,14 +597,31 @@ def wrapped_track_greedy(kymo, channel, min_length, **kwargs):
     def create_algorithm_sliders(self):
         import ipywidgets
 
-        return ipywidgets.VBox(
+        slider_box = ipywidgets.VBox(
             [
                 self._add_slider(key, parameter)
                 for key, parameter in self._algorithm_parameters.items()
                 if parameter.ui_visible
             ]
         )
 
+        image = self._kymo.get_image(self._channel)
+        min_threshold = np.percentile(image, 80)
+
+        threshold_slider_index = list(self._algorithm_parameters.keys()).index("pixel_threshold")
+        slider_box.children[threshold_slider_index].children[0].observe(
+            lambda change: self._set_label(
+                "warning",
+                ""
+                if change["new"] > min_threshold
+                else f"Tracking with threshold of {change['new']} maybe be slow.",
```

Comment:
`Tracking with threshold of 1 maybe be slow` sounds a bit odd to me because of the double `be`. Maybe something like `Tracking with threshold of 1 may be slow`

Reply:
oops, typo. what you suggest is what I meant to put